tests/opentelemetry-test-utils/src/opentelemetry/test/test_base.py

```diff
@@ -56,6 +51,10 @@ def tearDownClass(cls):
 
     def setUp(self):
         self.memory_exporter.clear()
+        reset_metrics_globals()
+        result = self.create_meter_provider()
+        self.meter_provider, self.memory_metrics_reader = result
```

Comment:
ik this was copy pasted from above, but it doesn't look like `result` is used
```suggestion
        self.meter_provider, self.memory_metrics_reader = self.create_meter_provider()
```